lib/rubocop/cop/lint/void.rb

```diff
@@ -102,6 +104,17 @@ def check_for_defined(node)
 
           add_offense(node, :expression, format(DEFINED_MSG, node.source))
         end
+
+        # `initialize` and setter methods is not return value of last
+        # expression.
+        def in_void_context_method?(node)
+          parent = node.parent
+          return false unless parent
+          method = parent.children.first
+          parent.def_type? &&
```

Comment:
Yes. Both `def` and `defs` have been on my list for some time. They're probably next. 🙂